docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -17,17 +17,25 @@ When using the DMT, bear the following in mind:
 ** The source cluster must be running and populated with required data
 ** The application must be using the source cluster successfully before migration
 ** Before starting the migration, remember to shutdown any applications that are using the source cluster
-** For certain versions, you must add the provided extension JAR to your class path and restart the cluster if you want to migrate Replicated map metadata and/or CP data structures. For further information on the affected versions and use of the extension JAR, see the xref:migrate:dmt-source-cluster-extension.adoc[] topic
+** For certain versions, you must add the provided extension JAR to your class path and restart the cluster if you want to migrate Replicated map metadata and/or CP data structures. 
 +
-If needed, but not done, Replicated map metadata and/or CP data structure are not migrated. For further information on adding the extension JAR to your classpath, refer to the https://docs.oracle.com/javase/6/docs/technotes/tools/windows/classpath.html[Oracle documentation,window=_blank].
+For further information on the affected versions and use of the extension JAR, see the xref:migrate:dmt-source-cluster-extension.adoc[] topic. For further information on adding the extension JAR to your classpath, refer to the https://docs.oracle.com/javase/6/docs/technotes/tools/windows/classpath.html[Oracle documentation,window=_blank].
++
+If needed, but not done, the following occurs:
+
+*** The migration cluster attempts to migrate Replicated map metadata unless you have disabled the migration of Replicated map metadata in your configuration, as described in xref:migrate:dmt-connection-config.adoc#replicatedmap-metadata-migration[ReplicatedMap Metadata Migration]. 
++
+If you have not added the extension JAR or disabled the migration of Replicated map metadata, the migration fails with an error about the missing API contained in the extension JAR.
+
+*** CP data structure are not migrated. 
```

Comment:
since we say if needed, lets change here as well. an extension jar is needed if they want to migrate cp structures. 

So, "if needed" means they selected cp structures to migrate.

if they selected cp structures to migrate, but haven't added the extension jar to the classpath the behaviour is not "CP data structure are not migrated."

it's failing the migration with an error about the missing CP APIs that are in the extension jar or 5.3.7+ cluster version.